### DISCUSSION TITLE
Why does undici use `ReadableStream` from `stream/web` if it is a Node library?

### DISCUSSION THREAD

**Discussion by dlarocque** [nodejs/undici#3409] [Q&A] [answered]
In Node, there exist different interfaces for `ReadableStream`:
- [`node/globals.d.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/globals.d.ts#L273) seems to implement https://nodejs.org/api/stream.html#class-streamreadable
- [`node/stream/web.d.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/stream/web.d.ts#L146) seems to implement https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream

In undici, we use the web interface here: https://github.com/nodejs/undici/blob/main/types/fetch.d.ts#L7. 
Why don't we use the Node interface?

**Comment by Ethan-Arrowood** [comment, marked as answer]
Fetch is a browser API and Undici implements it as close to specification as possible (meaning it uses Web `ReadableStream`). 

Other methods in Undici use the Node.js Stream `Readable`. 

Node.js has support for many Web Apis (specified by WHATWG or W3C)

**Reply by dlarocque** [reply]
Thanks for the answer!
